node/flatpak_node_generator/providers/yarn.py

```diff
@@ -49,7 +49,7 @@ def parse_package_section(self, lockfile: Path, section: List[str]) -> Package:
         name_line = name_line[:-1]
 
         name = self.unquote(name_line.split(',', 1)[0])
-        name, _ = name.rsplit('@', 1)
+        name, name_version = name.rsplit('@', 1)
```

Comment:
Bit of a nit, but the name here is a bit confusing, maybe `version_constraint` since it can include the `^` syntax and such?